hack/make-rules/verify/codegen.sh

```diff
@@ -17,39 +17,31 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ -n "${TEST_WORKSPACE:-}" ]]; then # Running inside bazel
-  echo "Validating codegen files..." >&2
-elif ! command -v bazel &> /dev/null; then
-  echo "Install bazel at https://bazel.build" >&2
-  exit 1
-else
-  (
-    set -o xtrace
-    bazel test --test_output=streamed @io_k8s_test_infra//hack:verify-codegen
-  )
-  exit 0
-fi
-
-SCRIPT_ROOT=$PWD
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../.." && pwd -P)"
+cd $REPO_ROOT
 
-DIFFROOT="${SCRIPT_ROOT}/prow"
-TMP_DIFFROOT="${TEST_TMPDIR}/prow"
+# place to stick temp binaries
+BINDIR="${REPO_ROOT}/_bin"
 
+DIFFROOT="${REPO_ROOT}/prow"
+TMP_DIFFROOT="$(TMPDIR="${BINDIR}" mktemp -d "${BINDIR}/verify-codegen.XXXXX")"
 mkdir -p "${TMP_DIFFROOT}"
```

Comment:
nit: I think this is handled my `mktemp -d` now.

We should also clean up `TMP_DIFFROOT` when we're done with it. As is, running this locally will continually grow the size of the BINDIR.

Reply:
good point, done